runelite-api/src/main/java/net/runelite/api/HitsplatID.java

```diff
@@ -30,10 +30,12 @@ public final class HitsplatID
 	public static final int BLOCK_OTHER = 13;
 	public static final int DAMAGE_ME = 16;
 	public static final int DAMAGE_OTHER = 17;
-	public static final int POISON = 2;
+	public static final int POISON = 65;
```

Comment:
Everywhere I checked poison was actually 65, not sure if they moved them around at some point.